packages/pyright-internal/src/tests/typeEvaluator1.test.ts

```diff
@@ -302,6 +302,18 @@ test('TypeNarrowingIsNoneTuple1', () => {
     TestUtils.validateResults(analysisResults, 0);
 });
 
+test('TypeNarrowingIsNoneNamedTuple1', () => {
+    const analysisResults = TestUtils.typeAnalyzeSampleFiles(['typeNarrowingIsNoneNamedTuple1.py']);
+
+    TestUtils.validateResults(analysisResults, 0);
+});
+
+test('TypeNarrowingIsNoneNamedTuple2', () => {
+    const analysisResults = TestUtils.typeAnalyzeSampleFiles(['TypeNarrowingIsNoneNamedTuple2.py']);
```

Comment:
The file name here must be lowercase. Otherwise the test will fail on OSes (like Linux) that have case-sensitive file systems.